src/feelpp/benchmarking/reframe/report.py

```diff
@@ -97,31 +130,43 @@ def buildPerf(self, df):
 
 
     def buildSpeedup(self, df):
+        """
+        Calculates the speedup for each extracted performance value,
+        based on the value with the lowest CPU number
+        """
         self.ref_speedup = df['num_tasks'].min()
-        mainRefs = self.df_perf[self.df_perf['num_tasks'] == self.ref_speedup]
-        print("Main performance variables:\n", mainRefs.to_markdown())
 
+        mainRefs = self.df_perf[self.df_perf['num_tasks'] == self.ref_speedup]
         if self.partial:
             partRefs = self.df_partialPerf[self.df_partialPerf['num_tasks'] == self.ref_speedup]
-            print("Partial performances variables\n", partRefs.to_markdown())
 
         for k, t in enumerate(df['num_tasks'].unique()):
             mainIndex = 0
             partIndex = 0
             for perf in df['perfvars'][k]:
```

Comment:
See above

Reply:
to do while refracting in [#72](https://github.com/feelpp/benchmarking/issues/72)